SpatialGDK/Extras/schema/rpc_payload.schema

```diff
@@ -10,5 +10,4 @@ type UnrealRPCPayload {
     uint32 offset = 1;
     uint32 rpc_index = 2;
     bytes rpc_payload = 3;
-    option<TracePayload> rpc_trace = 4;
```

Comment:
We don't need this to be visible in inspector / snapshots and is not a requirement of squid. Thoughts?

Reply:
I like if it is not needed